plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkPreferences.java

```diff
@@ -57,7 +61,41 @@ static IEclipsePreferences getPreferenceNode() {
 
   @Override
   public void initializeDefaultPreferences() {
-    getPreferenceStore().setDefault(CLOUD_SDK_MANAGEMENT,
-        CloudSdkManagementOption.AUTOMATIC.name());
+    initializeDefaultPreferences(getPreferenceStore());
+  }
+
+  @VisibleForTesting
+  public void initializeDefaultPreferences(IPreferenceStore preferences) {
+    preferences.setDefault(CLOUD_SDK_MANAGEMENT, CloudSdkManagementOption.AUTOMATIC.name());
+    if (CloudSdkManager.isManagedSdkFeatureEnabled()
+        && preferences.isDefault(CLOUD_SDK_MANAGEMENT)) {
```

Comment:
Shouldn't this be `preferences.getString(CLOUD_SDK_MANAGEMENT).isEmpty()`? I think `isDefault()` is basically `.getString(CLOUD_SDK_MANAGEMENT).equals("AUTOMATIC")` here.

Reply:
To avoid the possibly ambiguity in `isDefault()`, I can move the `setDefault()` and instead check if it's set.